web/satellite/src/components/buckets/BucketArea.vue

```diff
@@ -7,18 +7,16 @@
         <div class="buckets-overflow" v-else>
             <div class="buckets-header">
                 <p>Buckets</p>
-                <SearchArea/>
+                <HeaderComponent class="buckets-header-component" placeholder="Buckets" :search="fetch"/>
```

Comment:
Should this placeholder be passed in as a prop?

Reply:
yeah, we want to have different placeholders in our header component. So we use it as a prop